submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/request/SubmissionRequestCleanIndexer.kt

```diff
@@ -63,6 +64,7 @@ class SubmissionRequestCleanIndexer(
         newFiles: FilesRecords,
         current: ExtSubmission,
     ): Triple<Int, Int, Int> {
+        val reusedIdx = AtomicInteger(0)
```

Comment:
if you noticed we return conflict and deprecated values so latter we know what is the amount of files to delete so we can acurrectly calculate the progress of procress. With reused files we are not so when we are actually unreleased we do not know how many files are pending.  


Not saying this need to be included in this ticket which was already a huge effort but I think having precise estimation of pending activities is something we do not want to lose.  Create a ticket to address that after this?

Reply:
you're right, I added the changes 👍